### DISCUSSION TITLE
Problem with the model when sending from MoBi to PKSim

### DISCUSSION THREAD

**Discussion by MaLuisaEti** [Open-Systems-Pharmacology/Forum#1557] [Models] [answered]
Hi! 
I have developed a PBPK model and modified it in MoBi, making one of the parameters (specific blood flow rate in Kidneys) drug´s concentration-dependent. However, when I sent it to PKSim for population simulation, I noticed that the model doesn't fit as well. It appears that PKSim might not be incorporating some of the modifications I included in MoBi, particularly the concentration-dependent parameter. I would like to know if there is something I should take into account when exporting the simulation to PKSim that I am currently overlooking. 

To export the simulation from MoBi to PKSim, I tried the following two options: 1) I used the tool in MoBi that appears when right-clicking and selecting "Send to PKSim for Population Simulation." 2) I saved the MoBi simulation in PKML format and then opened it in PKSim. Both procedures yielded the same result.

Thank you in advance for your response. 
Regards.
Ma Luisa

**Comment by msevestre** [comment, marked as answer]
@MaLuisaEti This is the right way of doing this
However, your population also has values for the kidney blood flow (as this is an individual parameter for which a distribution is defined in the database). That means that it is **CRUCIAL** that you remove this parameter from your population before simulating
HOW: 

1. Export to csv, 
2. remove the column, 
3. create a new population-based on the modified csv
4. enjoy life

**Reply by MaLuisaEti** [reply]
Thank you very much @msevestre . I'll try this. 

Have a great weekend!

Ma Luisa
